master/buildbot/reporters/github.py

```diff
@@ -135,12 +142,24 @@ def send(self, build):
         else:
             issue = None
 
-        if project:
+        if "/" in project:
             repoOwner, repoName = project.split('/')
-        else:
+        # Given we have a repository define try to get the owner and name
+        elif sourcestamps[0]['repository']:
```

Comment:
Please install codecov chrome extension or look at the report.
We require that those kind of changes are unit tested before they can be included in the code